Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py

```diff
@@ -256,6 +267,246 @@ def get_query_by_name(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
     )
 
 
+def get_companies(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
+    count, companies = octox.get_companies(
+        page=args.get("page", 1),
+        search=args.get("search", ""),
+        size=args.get("size", 20),
+    )
+
+    return CommandResults(
+        outputs_prefix="OctoxLabs.Companies",
```

Comment:
```suggestion
        outputs_prefix="OctoxLabs.Company",
```
You should do it for the rest of the commands as well (domains, users) etc.